api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/controller/EditRequestDispatchController.java

```diff
@@ -43,71 +51,101 @@
  * by the N3 editing system.  It will examine the request parameters, determine
  * which form to use, execute a EditConfiguration setup, and evaluate the
  * view indicated by the EditConfiguration.
- * 
```

Comment:
@hudajkhan : there are a lot of whitespace changes that clutter this pull-request. Would you be able to limit the updates to the functional bits?